src/storage/src/hummock/iterator/forward_user.rs

```diff
@@ -46,6 +43,19 @@ pub type UserIteratorPayloadType<
     I,
 >;
 
+pub enum DirectedUserIterator {
```

Comment:
Can we move the code to mod.rs ? (DirectedUserIterator, DirectedUserIteratorBuilder, Payload...)

it looks like some generic code and use in backward_user.rs